keyboards/geekboards/macropad_v2/bootloader_defs.h

```diff
@@ -0,0 +1,7 @@
+/* Address for jumping to bootloader on STM32 chips. */
```

Comment:
This file may not be needed any more due to the existence of:
platforms/chibios/GENERIC_STM32_F072XB/configs/bootloader_defs.h

Reply:
removed it